Flex space

AdmirJr opened the pull request:
Allow modification of environmental variables during the simulation.

AdmirJr committed:
daisy world added

AdmirJr committed:
skeleton config updated

AdmirJr committed:
fixed some issues

ohagen committed:
Refine comments in input_creation.R

Updated comments for clarity

ohagen commented on R/simulation.R:
good catch @AdmirJr

ohagen replied:
no plotting triggered directly on init_ancestor_distribution... so this oldpar was redundant.

ohagen committed:
Refactor ecology section headers and comments

Updated section headers and comments for clarity and better vocabulary for Earth Systems
